index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>my-project</title>
+  </head>
+  <body>
+    <div id="app"></div>
```

Comment:
This id is how vue knows where to render the app.  It corresponds to the `#app` on line 5 of main.js.